src/Effect.js

```diff
@@ -147,8 +147,15 @@ class Effect extends EventEmitter {
    * @return {number} attribute modified by effect
    */
   modifyAttribute(attrName, currentValue) {
-    const modifier = (this.modifiers.attributes[attrName] || (_ => _)).bind(this);
-    return modifier(currentValue);
+    let modifier = _ => _;
```

Comment:
It's the identity function, could also be written as `x => x`. It returns whatever it gets. Handy for two reasons: attribute modification in effects works as a reducer, so if nothing else, the effect is just going to keep the attribute the same. If it were undefined that could cause issues (the infamous `undefined is not a function`.